h/security/policy/bearer_token.py

```diff
@@ -21,6 +19,11 @@ class BearerTokenPolicy(IdentityBasedPolicy):
     def __init__(self):
         self._identity_cache = RequestLocalCache(self._load_identity)
 
+    @staticmethod
+    def handles(request) -> bool:
+        """Return True if this policy applies to `request`."""
+        return is_api_request(request)
```

Comment:
All API sub-policies (currently `BearerTokenPolicy` and `AuthClientPolicy`) have this `handles(request)` method that the top-level `APIPolicy` uses to decide which sub-policy to delegate to. `AuthClientPolicy` already had this method, adding it to `BearerTokenPolicy` as well.